app/notifiers/publishing_api_notifier.rb

```diff
@@ -0,0 +1,14 @@
+class PublishingApiNotifier
+  def self.publish
+    new.publish
```

Comment:
We could get rid of this method by calling `PublishingApiNotifier.new.publish` in the rake task.